examples/02-custom-vpc-cidr-no-nodes.yaml

```diff
@@ -8,6 +8,8 @@ metadata:
   name: cluster-2
   region: eu-north-1
 
-vpc: {cidr: 10.10.0.0/16}
+vpc: 
+  cidr: 10.10.0.0/16
+  extraCIDRs: [ 54.168.0.0/16 ] 
```

Comment:
Maybe we should use `[ 10.20.0.0/16, 10.30.0.0/16, 10.40.0.0/16, 10.50.0.0/16 ]`, as that works out of the box?